packages/interactor/karma.conf.js

```diff
@@ -0,0 +1,71 @@
+module.exports = config => {
+  config.set({
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      {
+        pattern: require.resolve('regenerator-runtime/runtime'),
```

Comment:
I forgot, we should also add `regenerator-runtime` as an explicit dependency. It must be coming in from some other package right now.

Reply:
Done